modules/mobile/bin/health_checks/Gemfile

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
I haven't yet added a Gemfile.lock because that seems to be git ignored in this directory. As mentioned, I don't think this is the ideal location for this to live anyhow.